Documentation/BuildInstructionsWindows.md

```diff
@@ -0,0 +1,123 @@
+
+WSL Version 2 requires Windows 10 version 2004 or higher, with OS Build 19041 or greater. Here is a guide on how to
+[get WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+
+Once installed, you will need to make sure the distribution you want to use (and the new default) is using Version 2:
+- `wsl -l -v` lists distros and versions,<br/>
+- `wsl --set-version <distro> <version>` is used to convert a distro to another version, and<br/>
+- `wsl --set-default-version 2` will set the default version for all new distros (if desired.)<br/>
+
+## Setting up build tools
+
+Please see the general build instructions for a list of tools you need to install in your WSL Linux environment. As a
+special exception you should _not_ install QEMU in the Linux environment and instead use the instructions from the next
+section to set up QEMU on your host system.
+
+## Setting up QEMU
+
+- Grab the latest QEMU binaries from [here](https://www.qemu.org/download/#windows) and install them.
+
+- Locate the executable `qemu-system-i386.exe` in WSL.
+By default this will be located at `/mnt/c/Program Files/qemu/qemu-system-i386.exe`.
+
+- Set the `SERENITY_QEMU_BIN` environment variable to the location above. For example: \
+`export SERENITY_QEMU_BIN='/mnt/c/Program Files/qemu/qemu-system-i386.exe'`
+
+- Locate the _Windows_ path to the SerenityOS disk image, as native QEMU will be accessing it via the Windows filesystem.
+  If your build tree is located in the WSL2 partition, this will be accessible under the `\\wsl$` network file share
+  (see [notes below](#note-on-filesystems)).
+
+- Set the `SERENITY_KERNEL_CMDLINE` environment variable to disable VirtIO support (Because it is currently broken on
+  native windows QEMU):
+`export SERENITY_KERNEL_CMDLINE="disable_virtio"`
+
+- Set the `SERENITY_DISK_IMAGE` environment variable to the full path of the SerenityOS disk image file from above.
+  For example: `export SERENITY_DISK_IMAGE='\\wsl$\Ubuntu-20.04\home\username\serenity\Build\i686\_disk_image'`
+
+- `ninja run` as usual.
+
+### Hardware acceleration
+
+The steps above will run QEMU in software virtualisation mode, which is very slow.
+QEMU supports hardware acceleration on Windows via the [Windows Hypervisor Platform](https://docs.microsoft.com/en-us/virtualization/api/) (WHPX), a user-mode virtualisation API that can be used alongside Hyper-V.
+This is important to note as WSL2 itself runs on top of Hyper-V, which conflicts with other acceleration technologies
+such as Intel HAXM.
+
+To run SerenityOS in a WHPX-enabled QEMU VM:
+
+- If you have not already done so, enable Windows Hypervisor Platform, either using "Turn Windows features on or off",
+  or by running the following command in an elevated PowerShell session: \
+`dism /Online /Enable-Feature /All /FeatureName:HypervisorPlatform`
+
+- Specify QEMU acceleration option: \
+`export SERENITY_EXTRA_QEMU_ARGS="-accel whpx"`
```

Comment:
We should mention that with QEMU 6.x, `kernel-irqchip=off` is required 

Should also use SERENITY_VIRT_TECH_ARG, as that's what all the other accel options (kvm, hvf etc) end up being placed in

```suggestion
`export SERENITY_VIRT_TECH_ARG="-accel whpx,kernel-irqchip=off"`
```

Reply:
I've changed that to `SERENITY_VIRT_TECH_ARG`. As for `kernel-irqchip=off` - turns out we're already mentioning that twice but there's so much text it's quite easy to miss.